app/helpers/payments_helper.rb

```diff
@@ -0,0 +1,9 @@
+module PaymentsHelper
+
+  PAYMENT_METHODS = Payment.payment_methods
+
+  def get_payment_methods
+    PAYMENT_METHODS.keys.map {|payment_method| [payment_method.titleize,payment_method]}
```

Comment:
PAYMENT_METHODS should store the options array itself. Here each time `get_payment_methods` called, it will create 2D array for options